simtools/simtel/simtel_runner_camera_efficiency.py

```diff
@@ -82,16 +82,7 @@ def _make_run_command(self, **kwargs):  # pylint: disable=unused-argument
         pixel_diameter = self._telescope_model.camera.get_pixel_diameter()
 
         # Processing focal length
-        focal_length = 0.0
-        try:
-            focal_length = self._telescope_model.get_parameter_value_with_unit(
-                "effective_focal_length"
-            )[0]
-        except IndexError:
-            pass
-        if focal_length == 0.0:
-            self._logger.warning("Using focal_length because effective_focal_length is 0")
```

Comment:
the warning did not make it into the other function. Perhaps a good idea to have a warning there too if we keep this implementation.

Reply:
OK, done.